lib/Doctrine/Common/Reflection/StaticReflectionParser.php

```diff
@@ -153,8 +154,10 @@ protected function parse()
                     $docComment = $token[1];
                     break;
                 case T_CLASS:
-                    $this->docComment['class'] = $docComment;
-                    $docComment = '';
+                    if ($last_token !== T_PAAMAYIM_NEKUDOTAYIM) {
```

Comment:
Would it make sense to add a comment to this value? It isn't really clear why or what the the value equates too.